install_devconsole/consoledeveloper.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set +x
+
+oc apply -f ./yamls/unmanage.yaml
+oc scale --replicas 0 deployment console-operator --namespace openshift-console-operator
+oc scale --replicas 0 deployment console --namespace openshift-console
+oc apply -f ./yamls/redeploy-console-operator.yaml
+#It takes time to get the pod in running state
+while [ "$(oc get pods --field-selector=status.phase=Running -n openshift-console-operator)" == "No resources found." ]
+do
+    sleep 1s
+done
+
+oc scale --replicas 1 deployment console --namespace openshift-console
+#Delete the already existing pod in the openshift-console namespace
+#Because it's a Deployment, Kubernetes will automatically recreate the pod and pull the latest image.
+#Have also updated the image pull policy to Always in the yamls/redeploy-console-operator.yaml
+
+CONSOLE_POD="$(oc get pods -o=name -n openshift-console | cut -d'/' -f2- | cut -f 1 -d "-" | head -n 1)"
+CONSOLE_POD_NAME="$(oc get pods -o=name -n openshift-console | cut -d'/' -f2- | cut -d'-' -f1- | head -n 1)"
+if echo "${CONSOLE_POD}" == "console";then
+    oc delete pod ${CONSOLE_POD_NAME} -n openshift-console
```

Comment:
These would have got deleted when 
```
oc scale --replicas 0 deployment console --namespace openshift-console
```
was done?

Reply:
yeah..but regarding this - If you are already using the  quay.io/redhat-developer/console:master-next image and the image has been updated, the deployment doesn't naturally pull the updated image. So to be really sure that it is the new image we can delete the existing one. Because it's a Deployment, Kubernetes will automatically recreate the pod and pull the latest image.